docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf

```diff
@@ -143,8 +146,10 @@ references_clause ::= 'REFERENCES' table_name [ ( column_name { ',' column_name
 
 key_action ::= 'NO' 'ACTION' | 'RESTRICT' | 'CASCADE' | 'SET' 'NULL' | 'SET' 'DEFAULT' ;
 
-storage_parameter ::= name [ '=' value] ;
+storage_parameter ::= param_name [ '=' param_value] ;
```

Comment:
`param_name` and `param_value` don't have accompanying rules.  Can you create them?